static/src/javascripts/projects/common/modules/experiments/tests/politics-weekly-treat.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+import template from 'lodash/utilities/template';
+import treatHtml from 'raw-loader!journalism/views/politicsWeeklyTreat.html';
+import config from 'lib/config';
+import { getBreakpoint } from 'lib/detect';
+
+const runTest = function(variant: string): () => void {
+    return () => {
+        const headlinesContainer = document.querySelector(
+            '.facia-page #headlines'
+        );
+
+        if (headlinesContainer) {
+            const treats = headlinesContainer.querySelector(
+                '.treats__container'
+            );
+            if (treats) {
+                const newTreat = template(treatHtml, { variant });
+                treats.innerHTML = newTreat;
+            }
+        }
+    };
+};
+
+const trackClick = function(complete: () => void) {
+    const treat = document.querySelector('.politics-weekly-treat');
+    if (treat) {
+        treat.onclick = () => {
+            complete();
+        };
+    }
+};
+
+const trackImpression = function(track: () => void) {
+    const treat = document.querySelector('.politics-weekly-treat');
+    if (treat) {
+        const observer = new window.IntersectionObserver(
```

Comment:
`IntersectionObserver` is missing from quite a lot of browsers, notably all versions of Internet Explorer and Safari. We are not currently requesting it using polyfill.io. It's quite large at 21KB (although would be smaller when minified / gzipped). Will this test be seen by all users?

Reply:
I thought it's been added a while ago.

Reply:
[Indeed](https://github.com/guardian/frontend/commit/b259aaba474b6c25321f8304e88404c09772814f#diff-fd847ba59a0d17dcf4d298f0f0dfda75)

Reply:
Aha! Must have missed that, sorry 🙇